katas/main/src/pokerhands/adriel/HandKickersEvaluator.cs

```diff
@@ -0,0 +1,30 @@
+namespace PokerHands.HandsKickersEvaluator.Adriel;
+
+using Cards.Adriel;
+using Hands.Adriel;
+
+public class HandKickersEvaluator
```

Comment:
So "Kickers" are groups?

Reply:
Kickers are the cards used to break a tie when both hands have the same Ranking and card values